pkg/git/kube.go

```diff
@@ -17,23 +17,22 @@ limitations under the License.
 package git
 
 import (
+	"fmt"
 	"strings"
 
 	"gopkg.in/src-d/go-git.v4/plumbing"
 	"gopkg.in/src-d/go-git.v4/plumbing/object"
 )
 
-const (
-	KubernetesCommitPrefix         = "Kubernetes-commit: "
-	ancientSyncCommitSubjectPrefix = "sync(k8s.io/kubernetes)"
-)
-
-// kubeHash extracts kube commit from commit message
-func KubeHash(c *object.Commit) plumbing.Hash {
+// SourceHash extracts kube commit from commit message
+// The baseRepoName default to "kubernetes".
+func SourceHash(c *object.Commit, baseRepoOrg, baseRepoName string) plumbing.Hash {
 	lines := strings.Split(c.Message, "\n")
+	sourceCommitPrefix := strings.Title(baseRepoName) + "-commit: "
+	ancientSyncCommitSubjectPrefix := fmt.Sprintf("sync(%s/%s)", baseRepoOrg, baseRepoName)
```

Comment:
This ancientSyncCommitSubjectPrefix logic can go away. Then we only need the `commitMsgTag`, i.e. `Kubernetes-commit` in the kube case. Makes it much simpler.

Reply:
the flag for `collapsed-kube-commit-mapper` can then be `commit-msg-tag` as well. Makes it much clear why we need that info. Now it is surprising that the tool needs the repo and org.

Reply:
well that flag will need to be passed to the scripts as well, right? not sure if we need a new flag :)

Reply:
we can construct the commit msg tag in util.sh (maybe we do that already, because we need it for the git-filter-branch call).

Reply:
that is actually good catch, I haven't replaced Kubernetes-commit in util.sh

Reply:
@sttts i fixed util.sh to pass the proper commit message via commit_msg_tag... I would go get rid of ancientSyncCommitSubjectPrefix as follow-up, because the number of changes in this PR are getting big, so we should probably start testing this and we can improve later?